Sources/Algorithms/Unique.swift

```diff
@@ -14,33 +14,57 @@
 //===----------------------------------------------------------------------===//
 
 extension Sequence where Element: Hashable {
-  /// Returns an array with only the unique elements of this sequence, in the
-  /// order of the first occurence of each unique element.
+  /// Returns an array with only  unique elements of this sequence, in the
+  /// order of the first occurrence of each unique element.
+  ///
+  /// In this example, `uniqued()` is used to include only names which are
+  /// unique
```

Comment:
Thanks, the formatting is now mostly consistent. Make sure to include periods at the end of sentences, like here (also elsewhere.) There is still another closing brace that is incorrectly indented. There are also several places where you have two consecutive spaces in the comment or the code; please make sure to remove them. There are also places where there are missing spaces after punctuation: please make sure to add them.

A more general comment here: I don’t think it adds anything to describe the example here by saying that `uniqued` makes the array unique.